examples/streampay/zillabase/migrations/000002__create_balances.sql

```diff
@@ -31,7 +26,7 @@ CREATE VIEW all_user_transactions AS
   FROM
       user_transactions;
 
-CREATE MATERIALIZED VIEW streampay_balances AS
+CREATE ZVIEW streampay_balances AS
```

Comment:
Is it intentional to go with `ZVIEW` instead of `ZMVIEW` here since we cannot sink a non-materialized view anyway?

Reply:
actually, I meant `ZMVIEW`